application/app.py

```diff
@@ -133,22 +134,28 @@ def test_object(self):
                             frame_start = frame1,\
                             num_frames = nframes)
 
-    # for the run button
-    def run(self):
+    # for the runAnalysis button
+    def runAnalysis(self):
         """
         Runs TrafficIntelligence trackers and support scripts.
         """
         #TODO: Make email some internal parameter.
-        remote.analysis(get_config_with_sections(get_config_path(), 'info', 'identifier'))
+        self.api.analysis(get_config_with_sections(get_config_path(), 'info', 'identifier'))
```

Comment:
This is gonna conflict, when you merge make sure this is `api` and not `self.api`.